src/types/any.rs

```diff
@@ -715,9 +717,9 @@ impl PyAny {
     /// }
     ///
     /// Python::with_gil(|py| {
-    ///     let class: &PyAny = Py::new(py, Class { i: 0 }).unwrap().into_ref(py);
+    ///     let class = Py::new(py, Class { i: 0 }).unwrap().into_bound(py).into_any();
     ///
-    ///     let class_cell: &PyCell<Class> = class.downcast()?;
+    ///     let class_cell: &Bound<'_, Class> = class.downcast()?;
     ///
     ///     class_cell.borrow_mut().i += 1;
```

Comment:
The variable `class_cell` maybe should be called `class_py` or `class_bound` instead?